source/index.txt

```diff
@@ -120,6 +121,14 @@ Upgrade {+driver-short+} Versions
 Learn what changes you might need to make to your application to upgrade driver versions
 in the :ref:`pymongo-upgrade` section.
 
+Migrate to the {+driver-async+} Driver
+------------------------------------
+
+In September 2024, MongoDB released the {+driver-async+} driver to unify {+driver-short+}
+and `Motor <https://www.mongodb.com/docs/drivers/motor/>`__, the asynchronous
+MongoDB driver for Python. Learn how to migrate from {+driver-short+} or Motor
```

Comment:
We should be clear that the new Async API is a replacement for Motor specifically and that any existing users of PyMongo will only have to migrate if they want to switch to async.